Add support for adding new site(s) to site replication (#13696)
Minio-io/minio: docs/site-replication/README.md

1. Initially, only **one** of the sites being added for replication may have data. After site-replication is successfully configured, this data is replicated to the other (initially empty) sites. Subsequently, objects may be written to any of the sites, and they will be replicated to all other sites.
2. Only the **LDAP IDP** is currently supported.

> 3. At present, all sites are **required** to have the same root credentials.

Reviewer: ```suggestion
3. All sites **must** have the same root credentials.
```

> 4. At present it is not possible to **remove a site** from a set of replicated sites.

Reviewer: We shouldn't talk about this as a limitation - this is totally expected that they can't remove a site. 

```suggestion
4.  **removing a site** is not allowed from a set of replicated sites once configured. 
```

> 5. If using [SSE-S3 or SSE-KMS encryption via KMS](https://docs.min.io/docs/minio-kms-quickstart-guide.html "MinIO KMS Guide"), all sites are required to have access to the same KES keys. This can be achieved via a central KES server or multiple KES servers (say one per site) connected to a central KMS server.

Reviewer: ```suggestion
5. [SSE-S3 or SSE-KMS encryption via KMS](https://docs.min.io/docs/minio-kms-quickstart-guide.html "MinIO KMS Guide"), all sites **must**  have access to the same KMS keys. This can be achieved via a central KES server or multiple KES servers (say one per site) connected via a central KMS server.
```
